### PR TITLE
Fixes issue #724 #725 bug resolved in activities

### PR DESCRIPTION
**Pull request by Avjeet**
Fixes issue #724 #725 

Changes: 
added bind view in oscilloscope acitivity
and added snackbar code in sensordatalogger activity

### CODE REVIEW

**Comment by Avjeet** [comment]
@CloudyPadmal @asitava1998 mentors plz review the PR .

[app-debug.zip](https://github.com/fossasia/pslab-android/files/1748785/app-debug.zip)
here is the debug apk

File: app/src/main/java/org/fossasia/pslab/activity/SensorDataLoggerActivity.java [hunk 1]
```diff
@@ -141,7 +141,8 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 } else {
                     Snackbar snackbar = Snackbar.make(coordinatorLayout, "Device not connected", Snackbar.LENGTH_SHORT);
                     View snackBarView = snackbar.getView();
-                    snackbarTextView.setTextColor(Color.YELLOW);
+                    TextView snackBarTextView = (TextView) snackBarView.findViewById(android.support.design.R.id.snackbar_text);
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/activity/SensorDataLoggerActivity.java]: remove the global `snackBarTextView`

Comment (Avjeet) [reply app/src/main/java/org/fossasia/pslab/activity/SensorDataLoggerActivity.java]: changes done @viveksb007

**Comment by Avjeet** [comment]
@CloudyPadmal can u plz merge the PR so that I can send for other issue.

**Comment by CloudyPadmal** [comment]
@Avjeet you can send more PRs by working them in different branches! This PR needs to be approved by @viveksb007 ! :+1:

**Pull request merged**